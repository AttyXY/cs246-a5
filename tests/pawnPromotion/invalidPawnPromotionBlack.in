setup
+ k h1
+ K a8
+ p a2
= black
done
game human human
move a2 a1 p    
move a2 a1 k
move a2 a1 x
move a2 a1 R
