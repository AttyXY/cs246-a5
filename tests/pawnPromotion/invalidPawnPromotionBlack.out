WELCOME TO CHESS! PLEASE ENTER ONE OF THE FOLLOWING COMMANDS TO BEGIN:
- setup
- game [player1] [player2]

STARTING SETUP

8  --------
7  --------
6  --------
5  --------
4  --------
3  --------
2  --------
1  -------k
   abcdefgh

8  K-------
7  --------
6  --------
5  --------
4  --------
3  --------
2  --------
1  -------k
   abcdefgh

8  K-------
7  --------
6  --------
5  --------
4  --------
3  --------
2  p-------
1  -------k
   abcdefgh

STARTING GAME!

8  K-------
7  --------
6  --------
5  --------
4  --------
3  --------
2  p-------
1  -------k
   abcdefgh

Invalid pawn promotion type.
Invalid pawn promotion type.
Invalid pawn promotion type.
8  K-------
7  --------
6  --------
5  --------
4  --------
3  --------
2  --------
1  r------k
   abcdefgh

White is in check.
Final Score:
White: 0
Black: 0
THANKS FOR PLAYING CHESS!
